### PR TITLE
A3C Polishing

### PR DESCRIPTION
**Pull request by richardliaw**
Ready for review - 16 workers converges in about 15 +/- 3 minutes
Notes:
 - Added documentation for tensorboard
 - Added light performance numbers
 - Removed any dependence on `gym=0.7.4`

### CODE REVIEW

File: examples/a3c/driver.py [hunk 1]
```diff
@@ -10,16 +10,17 @@
 import six.moves.queue as queue
 import gym
 import sys
+import os
 from datetime import datetime, timedelta
-from misc import timestamp, Profiler
+from misc import *
```
Comment (robertnishihara) [inline examples/a3c/driver.py]: Can we avoid `import *`? I'd like to make this code PEP8 compliant later and the `import *` will make things a bit harder.

File: examples/a3c/runner.py [hunk 2]
```diff
@@ -138,7 +139,7 @@ def env_runner(env, policy, num_local_steps, summary_writer, render):
                 summary = tf.Summary()
                 for k, v in info.items():
                     summary.value.add(tag=k, simple_value=float(v))
-                summary_writer.add_summary(summary, policy.global_step.eval())
+                summary_writer.add_summary(summary, rollout_number) #policy.global_step.eval())
```
Comment (robertnishihara) [inline examples/a3c/runner.py]: should we remove this  comment?

File: examples/a3c/envs.py [hunk 3]
```diff
@@ -10,29 +10,52 @@
...
-    env = DiagnosticsInfo(env)
-    env = Unvectorize(env)
+    env = AtariProcessing(env)
+    env = Diagnostic(env)
     return env
 
-def DiagnosticsInfo(env, *args, **kwargs):
-    return vectorized.VectorizeFilter(env, DiagnosticsInfoI, *args, **kwargs)
+def _process_frame42(frame):
+    frame = frame[34:34+160, :160]
```
Comment (robertnishihara) [inline examples/a3c/envs.py]: should probably be formatted as `frame[34:(34 + 160), :160]` although i'm not 100% sure about the best way to do this one.

**Comment by richardliaw** [comment]
Timing is a bit slower now than previously benchmarked, likely due to #455.

**Pull request merged**